server/py/services/api/api/endpoints/model_endpoints.py

```diff
@@ -281,6 +281,7 @@ async def list_model_endpoints(
     top_level: bool = Query(False, alias="top-level"),
     tsdb_metrics_old: bool = Query(True, alias="tsdb_metrics"),  # TODO: remove in 1.11
     tsdb_metrics: bool = Query(True, alias="tsdb-metrics"),
+    metric_list: Optional[list[str]] = Query(None, alias="metric-list"),
```

Comment:
Doing this makes FastAPI support multiple instances of the `metric-list` parameter, for example to generate `["metric1","mertic2"]` you would need to call `...?metric-list=metric1&metric-list=metric2` which doesn't really seem reasonable from user perspective.
I suggest changing the parameter alias simply to `metric` or `metric-name` instead.